### DISCUSSION TITLE
Hyper app v2 effects repo 404 error

### DISCUSSION THREAD

**Discussion by razorback21** [jorgebucaran/hyperapp#1050] [Q&A] [answered]
Hi, 

I am following the tutorial in this link https://github.com/jorgebucaran/hyperapp/blob/main/docs/tutorial.md and I'm already at the effects section but I cannot continue because 'https://unpkg.com/@hyperapp/http' returns a 404 error.  I've tried searching but  I cannot find the repo URL. Please help.

**Comment by jorgebucaran** [comment, marked as answer]
The tutorial needs to be updated as that package is not available at this time (I'm not sure if I'll bring it back someday, but it's possible). Until then, please use https://github.com/okwolf/hyperapp-fx by @okwolf.

**Reply by razorback21** [reply]
Thank you very much.
